src/Form.php

```diff
@@ -304,9 +304,12 @@ public function onSubmit(\Closure $callback)
                 }
 
                 return $response;
-            } catch (\Atk4\Data\ValidationException $val) {
+            } catch (\Atk4\Data\ValidationException $e) {
                 $response = [];
-                foreach ($val->errors as $field => $error) {
+                foreach ($e->errors as $field => $error) {
+                    if (!$field) {
```

Comment:
as @mkrecek234 pointed out, this needs to be fixed in data, however, I belive, there is a usecase when field is empty when the source of the validation exception is unknown

Reply:
fixed in https://github.com/atk4/data/pull/942